napari/types.py

```diff
@@ -64,9 +64,9 @@
 # since it includes all valid arguments for np.array() ( int, float, str...)
 ArrayLike = Union[np.ndarray, 'dask.array.Array', 'zarr.Array']
 
-LayerTypeName = Literal[
-    "image", "labels", "points", "shapes", "surface", "tracks", "vectors"
-]
+# LayerTypeName = Literal[
+#     "image", "labels", "points", "shapes", "surface", "tracks", "vectors"
+# ]
```

Comment:
I don't understand why the tool would comment out this literal

Reply:
In general we define this in two places. `napari.types` and `npe2.types`. As `npe2` is released with `graph` in list and `graph` is not added here as proper PR is not merge yet. 

The main question is if we should keep double definitions. 

If Yes I could either add `graph` here or block npe2 bump. I do not know what is best option.

Reply:
hmm I'm not sure either - it feels more natural to define it in napari, can npe2 not use the napari type? To that end, maybe we should add `graph` here now and get the new npe2 in

Reply:
hm, the whole idea of npe2 is to be napari independent. But maybe we should make npe2 types fallback, defined only when napari is absent in the environment?

Reply:
I definitely prefer only defining this in one place.

I also prefer using the npe2 definition because as mentioned napari depends on npe2, but npe2 doesn't depend on napari. Also, the layer type name is only for the layer data 3-tuple, right? Which is mostly intended for plugin reader/writer contributions.

Regardless, this should be deleted instead of commented out. Or if we really want the commented out code, we should add a comment to explain why it's commented out.

Reply:
> Regardless, this should be deleted instead of commented out. Or if we really want the commented out code, we should add a comment to explain why it's commented out.

To make conflict in graph PR to resolve this. But if we decide to depend on npe2 then maybe just remove it.